smac/utils/io/traj_logging.py

```diff
@@ -209,7 +288,6 @@ def read_traj_aclib_format(fn: str, cs: ConfigurationSpace):
 
     @staticmethod
     def _convert_dict_to_config(config_list: typing.List[str], cs: ConfigurationSpace):
```

Comment:
I know this function already exists, but shouldn't one use `ConfigSpace.util.fix_types` here?

Reply:
We could, but `ConfigSpace.util.fix_types` assumes all [Categorical's](https://github.com/automl/ConfigSpace/blob/e51b1548ffb0c5df367d5107b1dcc676b3216868/ConfigSpace/util.pyx#L417-L419) and [Constant's](https://github.com/automl/ConfigSpace/blob/e51b1548ffb0c5df367d5107b1dcc676b3216868/ConfigSpace/util.pyx#L423-L425) are strings or am I wrong here?

Reply:
@mfeurer I think i'll leave it here at the least for backwards-compatibility, but I could imagine expanding the `fix_types` would be useful.

Reply:
Agreed, that's an issue. `fix_types` should inspect the types in the categorical and constants.